### DISCUSSION TITLE
UpperConfidenceBound acquisition function question

### DISCUSSION THREAD

**Discussion by dbernstein-symbio** [meta-pytorch/botorch#1198] [Q&A] [answered]
Hi Everyone, In the UCB acquisition function (https://botorch.org/api/_modules/botorch/acquisition/analytic.html#UpperConfidenceBound) does the posterior use the observation noise in computing the sigma term? Or not? I'm new to botorch so a bit hard for me to tell from the code.

Thanks,
Dave

**Comment by saitcakmak** [comment, marked as answer]
Hi Dave. It doesn't use the observation noise in the posterior. By default botorch posteriors do not include the observation noise unless 'observation_noise=True' is included in the 'model.posterior(...)' call.

**Reply by dbernstein-symbio** [reply]
Thanks. I am confused as to why not. I had the impression that the sigma term in the UCB should be that of the observation noise. Is that incorrect?

**Reply by saitcakmak** [reply]
My impression is that it should not include the observation noise. If you look at the GP-UCB [paper](https://arxiv.org/pdf/0912.3995.pdf), they do not add any noise term to the sigma. Which makes sense to me since in a heteroscedastic noise setting, an `x` with disproportionally large noise could overpower all other points even if we're certain about the corresponding function value, if we were to include the noise in sigma.

**Reply by dbernstein-symbio** [reply]
I see, thank you very much.
